dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -52,37 +68,64 @@ setup(){
     export PATH=$PATH:/usr/bin:/usr/local/go/bin
     export HOME=/home/airflow
     export AWS_REGION=us-west-2
+    export AWS_ACCOUNT_ID=$(cat ${json_file} | jq -r .aws_account_id)
     export AWS_ACCESS_KEY_ID=$(cat ${json_file} | jq -r .aws_access_key_id)
     export AWS_SECRET_ACCESS_KEY=$(cat ${json_file} | jq -r .aws_secret_access_key)
     export AWS_AUTHENTICATION_METHOD=$(cat ${json_file} | jq -r .aws_authentication_method)
     export ROSA_ENVIRONMENT=$(cat ${json_file} | jq -r .rosa_environment)
     export ROSA_TOKEN=$(cat ${json_file} | jq -r .rosa_token_${ROSA_ENVIRONMENT})
     export CLUSTER_NAME=$(cat ${json_file} | jq -r .openshift_cluster_name)
-    export ROSA_CLI_VERSION=$(cat ${json_file} | jq -r .rosa_cli_version)
-    if [[ ${ROSA_CLI_VERSION} == "master" ]]; then
-        git clone https://github.com/openshift/rosa
-	pushd rosa
-	make
-	sudo mv rosa /usr/local/bin/
-	popd
+    if [[ $INSTALL_METHOD == "osd" ]]; then
+        export OCM_CLI_VERSION=$(cat ${json_file} | jq -r .ocm_cli_version)
+        if [[ ${OCM_CLI_VERSION} == "master" ]]; then
+            git clone https://github.com/openshift-online/ocm-cli
+            pushd ocm-cli
+            sudo PATH=$PATH:/usr/bin:/usr/local/go/bin make
+            sudo mv ocm /usr/local/bin/
+            popd
+        fi
+        echo "Create new OSD access key.."
+        export ADMIN_KEY=$(aws iam create-access-key --user-name OsdCcsAdmin)
```

Comment:
Since we are validating ocm w/ an ocm whoami It might make sense for us to also do an equivalent aws whoami to ensure our creds are working. This can be done via
```
aws iam get-user | jq .User.UserName
```
ex:
# aws iam get-user | jq .User.UserName
"rzaleski"

If you don't filter to the user name you will expose the userid and arn which we may not want in the logs.